evap/settings.py

```diff
@@ -178,6 +183,10 @@
             'level': 'DEBUG',
             'propagate': True,
         },
+        'mozilla_django_oidc': {
+            'handlers': ['console'],
+            'level': 'DEBUG'
```

Comment:
I'm not sure anymore why the other loggers have `propagate: True`, but perhaps we want this here as well?

Reply:
also, have you decided to not log this to file and mail_admins, and if yes why?

Reply:
i updated the configuration to propagate and use file and mail_admins as well.